app/_landing_pages/tools.yaml

```diff
@@ -0,0 +1,135 @@
+                    A[Terraform] -->|Configures| B(Konnect)
+                end
+
+                subgraph "Management"
+                    B -->|Managed by| C(KIC)
+                    B -->|Configures Gateway entities| D(Admin API)
+                end
+
+                subgraph "Configuration Updates"
+                    F(decK) -->|Updates| B
+                    B -->|Updated by| E(Konnect API)
+                end
+
+  - row:
+      - column:
+          - type: structured_text
+            config:
+              header:
+                text: "Which Kong tools should I use?"
+              blocks:
+                - type: text
+                  text: "You can use multiple tools, depending on your use case:"
+                - type: unordered_list
+                  items:
+                    - "**Admin API:** Allows you to configure and manage Kong Gateway entities."
+                    - "**Konnect API:** Allows you to configure and manage Konnect-specific components, like Dev Portal or API Products."
+                    - "**decK:** Allows you to visualize configurations and apply changes in a declarative manner to Kong Gateway or Konnect."
+                    - "**Terraform:** Allows you to manage infrastructure as code and automated deployments to streamline setup and configuration of Konnect and Kong Gateway."
+                    - "**KIC:** Allows you to manage ingress traffic and routing rules."
+  
+  - row:
+      - column:
+          - type: header
+            config:
+              type: h2
+              text: "Learn more about Kong tools"
+              align: left
+  - row:
+    - column:
+      - type: card
+        config:
+          title: Admin API
+          icon:  https://docs.konghq.com/assets/images/icons/documentation/icn-admin-api-color.svg
+          cta: https://docs.konghq.com/gateway/latest/admin-api/
+    - column:
+      - type: card
+        config:
+          title: Konnect API
+          icon: https://docs.konghq.com/assets/images/icons/documentation/icn-references-color.svg
+          cta: https://docs.konghq.com/konnect/api/
+    - column:
+      - type: card
+        config:
+          title: decK
+          icon: https://docs.konghq.com/assets/images/icons/konnect/icn-cogwheel-nav.svg
+          cta: /deck/
+  - row:
+    - column:
+      - type: card
+        config:
+          title: Terraform
+          icon:  https://www.svgrepo.com/show/376353/terraform.svg
+          cta: https://docs.konghq.com/konnect/reference/terraform/
+    - column:
+      - type: card
+        config:
+          title: KIC
+          icon: https://docs.konghq.com/assets/images/icons/documentation/icn-kubernetes-color.svg
+          cta: https://docs.konghq.com/kubernetes-ingress-controller/latest/
+      
+  - row:
+      - column:
+          - type: header
+            config:
+              type: h2
+              text: Tutorials
+          - type: tutorial_list
+            config:
+              - tag: deck
```

Comment:
This won't work with the current implementation, but we can change that :)
```
config:
  - tag: # filters by tag, e.g. rate-limiting, etc. it only supports one, omit it for all tags
  - product: # filters by product, it only supports one , omit for all products
```

we can update to also support: `tool: # filter by tool, omit for all tools`

We could also make them accept more than one value, but we should use the plural for the key then

Reply:
I ended up implementing the changes I mentioned above ^